source/renderer/app/routes-config.js

```diff
@@ -30,6 +30,9 @@ export const ROUTES = {
     SETTINGS: '/wallets/:id/settings',
     UTXO: '/wallets/:id/utxo',
   },
+  VOTING: {
+    ADD: '/voting-catalyst/add',
```

Comment:
`ADD` should be renamed to `REGISTRATION`.
Route should be updated to '/voting/registration',